projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/query-management/read-query-management-field.component.ts

```diff
@@ -15,12 +15,14 @@ export class ReadQueryManagementFieldComponent extends AbstractFieldReadComponen
   public partyMessagesGroups: PartyMessagesGroup[];
   public query: QueryListItem;
   public showQueryList: boolean = true;
+  public caseId: string;
 
   constructor(private readonly route: ActivatedRoute) {
     super();
   }
 
   public ngOnInit(): void {
+    this.caseId = this.route.snapshot.params.cid;
```

Comment:
Should we use case notifier service to get the case id? @Daniel-Lam or @ronaldmansveld can you confirm?

Reply:
cid is already in the path params; I think getting it from CaseNotifier is an overkill